Hudl.Mjolnir/Command/Attribute/CommandInterceptor.cs

```diff
@@ -152,6 +152,16 @@ internal Command<TResult> CreateCommand<TResult>(IInvocation invocation)
                 throw new InvalidOperationException("Interface does not have [CommandAttribute]");
             }
 
+            if (newAttribute != null && newAttribute.IgnoreTimeout)
```

Comment:
This will take a look to see if the CommandAttribute has been set to ignore timeouts and then will pass this info onto a version of InvocationCommand that will deal with the ignored timeouts